cmd/ocm-addons/notify/cmd.go

```diff
@@ -0,0 +1,133 @@
+package notify
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/mt-sre/ocm-addons/cmd/ocm-addons/notify/list"
+	"github.com/mt-sre/ocm-addons/internal/cli"
+	"github.com/mt-sre/ocm-addons/internal/notification"
+	"github.com/mt-sre/ocm-addons/internal/ocm"
+
+	"github.com/apex/log"
+	"github.com/spf13/cobra"
+)
+
+func Cmd() *cobra.Command {
+	return generateCommand(run)
+}
+
+const _numArgs = 2
+
+func generateCommand(run func(*cobra.Command, []string) error) *cobra.Command {
```

Comment:
The example would be the same as the usage because there aren't any options to change the command behavior. But, I could probably explain the `NOTIFICATION_ID` expected input better.